pkg/util/podstore/type.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podstore
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// StoreInterface is a podStore interface to create local podStore for Flow Exporter and Flow Aggregator.
+type StoreInterface interface {
+	GetPodByIPAndTime(ip string, startTime time.Time) (*corev1.Pod, bool)
+	Run(stopCh <-chan struct{})
+}
```

Comment:
There isn't and likely won't be many types, perhaps just move this to podstore.go. And as this is the only interface of the package and the package name can reflect its purpose, it can be named as `Interface`, then there is no redundancy same when client using it: `podstore.Interface`.

Reply:
Updated. Thanks.